chart/k8gb/crd/k8gb.absa.oss_gslbs.yaml

```
@@ -492,3 +492,98 @@ spec:
     storage: true
     subresources:
       status: {}
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.19.0
+  name: zonedelegations.k8gb.absa.oss
+spec:
+  group: k8gb.absa.oss
```

```suggestion
  name: zonedelegations.k8gb.io
spec:
  group: k8gb.io
```

we might consider using new upcoming group for the new resource to avoid migrations